Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -103,72 +215,73 @@ def closePath(self):
 
         self.endPts.append(endPt)
 
-    def endPath(self):
+    def endPath(self) -> None:
         # TrueType contours are always "closed"
         self.closePath()
 
-    def addComponent(self, glyphName, transformation):
-        self.components.append((glyphName, transformation))
 
-    def _buildComponents(self, componentFlags):
-        if self.handleOverflowingTransforms:
-            # we can't encode transform values > 2 or < -2 in F2Dot14,
-            # so we must decompose the glyph if any transform exceeds these
-            overflowing = any(s > 2 or s < -2
-                              for (glyphName, transformation) in self.components
-                              for s in transformation[:4])
-        components = []
-        for glyphName, transformation in self.components:
-            if glyphName not in self.glyphSet:
-                self.log.warning(
-                    "skipped non-existing component '%s'", glyphName
-                )
-                continue
-            if (self.points or
-                    (self.handleOverflowingTransforms and overflowing)):
-                # can't have both coordinates and components, so decompose
-                tpen = TransformPen(self, transformation)
-                self.glyphSet[glyphName].draw(tpen)
-                continue
+class TTGlyphPointPen(TTGlyphBasePen, LogMixin, AbstractPointPen):
+    """
+    Pen used for drawing to a TrueType glyph.
```

Comment:
this docstring is identical to TTGlyphBasePen's, either you make it more specific (e.g. say it's a point pen, not just a pen), or if you keep the one from the base pen, you delete this one
```suggestion
    Point pen used for drawing to a TrueType glyph.
```

Reply:
I've added setting `_currentContourStartIndex` to the `init()` method of the point pen, which now in turn calls `super().init()`.

Reply:
... and moved the docstrings around.